### ISSUE TITLE
Typehint errors 

### ISSUE THREAD

**Issue by Teranode** [cyrildewit/laravel-page-view-counter#67]
- Laravel Page View Counter Version: 2.0
- Laravel Version: 5.5.dev
- PHP Version: 7.2.5
- Database Driver & Version: MariaDB 10.2

### Description:
Im not sure why but changing my cache to redis seems to be cause some typehinting errors:
```
Symfony\Component\Debug\Exception\FatalThrowableError: Type error: Return value of Tera\Comic\Models\Comic::getViews() must be of the type integer, string returned in /home/caligula/public_html/vendor/cyrildewit/laravel-page-view-counter/src/Viewable.php:56
```

```
public function getViews($period = null): int
```
to
```
public function getViews($period = null)
```
when i remove the typehint from "all" the functions then everything works fine otherwise i just use memcached again. It also works if i disable caching in the viewable config

### Steps To Reproduce:
Change cache to redis

**Comment by cyrildewit** [comment]
It might be possible that the returned value from redis is a string, so I have to convert them to an integer manually.

Could you test it by changing line 92 in `ViewableService` from `$cachedViewsCount = $this->cache->get($cacheKey);` to `$cachedViewsCount = (int) $this->cache->get($cacheKey);`

**Comment by Teranode** [comment]
It does fix the error , but messes up everything else in terms of ordering, however changing 
```
            if ($cachedViewsCount !== null) {
                return $cachedViewsCount;
            }
```
to
```
            if ($cachedViewsCount !== null) {
                return (int) $cachedViewsCount;
            }
```

works

**Comment by cyrildewit** [comment]
Thanks for reporting this bug! I pushed commit https://github.com/cyrildewit/laravel-page-view-counter/commit/a870c6aeca3c718a34271f081869301960cf848c with a fix to the `2.0` branch.

Run `composer update cyrildewit/laravel-page-view-counter` to get the latest version.
